DQM/DTMonitorModule/src/DTDataIntegrityTask.cc

```diff
@@ -1158,6 +1792,42 @@ void DTDataIntegrityTask::analyze(const edm::Event& e, const edm::EventSetup& c)
   nEventsLS++;
 
   LogTrace("DTRawToDigi|DTDQM|DTMonitorModule|DTDataIntegrityTask") << "[DTDataIntegrityTask]: preProcessEvent" <<endl;
+
+  if (checkUros){ //uROS starting on 2018
+  // Digi collection
+  edm::Handle<DTuROSFEDDataCollection> fedCol;
+  e.getByToken(fedToken, fedCol);
+
+  DTuROSFEDData fedData;
+  DTuROSROSData urosData;
+
+  if(fedCol.isValid()){
+    for(unsigned int j=0; j <fedCol->size();++j){
+
+    fedData = fedCol->at(j);
+    int fed = fedData.getfed(); //argument should be void
+
+//std::cout<<"FED" << fed << std::endl;
```

Comment:
Here and below: all this commented out cout's should be better removed. If needed, they could be wrapped into a ```ifdebug``` clause, or similar